client/src/login/actions.ts

```diff
@@ -23,24 +24,6 @@ export type LoginError = ReturnType<typeof loginError>;
 export const login = () => async dispatch => {
   dispatch(loginStart());
 
-  /**
-   * The Sign-In client object.
-   */
-  let auth2: any;
-
-  const googleAPILoaded: Promise<void> = new Promise(resolve => {
-    gapi.load('auth2', () => {
-      /**
-       * Retrieve the singleton for the GoogleAuth library and set up the
-       * client.
-       */
-      auth2 = gapi.auth2.init({
-          client_id: '634069824484-ch6gklc2fevg9852aohe6sv2ctq7icbk.apps.googleusercontent.com'
-      });
-      resolve();
-    });
-  });
-
```

Comment:
This seems like a good place for this logic to live... Why are you removing it?

Reply:
Yeah, I think the best option is probably to have 2 different actions, one to attempt to login in the background, and one that will only happen when the user clicks the login button

Reply:
Ah, okay. Well I don't know how to do that. So we'll talk on Wednesday